examples/step-63/step-63.cc

```diff
@@ -297,7 +323,13 @@ namespace Step63
   }
 
 
+  // @sect3{Right-hand Side and Boundary Values}
+
+  // The problem solved in this tutorial is an adaptation of Ex. ___
+  // found in ______ (how to cite?), namely, we add a hole in the middle
+  // of our domain.
 
```

Comment:
I'll need the book _Finite Element and Fast Iterative Solvers_ (Elman, Howard, Solvester, Wathen) to look up where exactly the example is and I'm not quite sure how to cite (but I can figure it out). But this can be done Monday.